### PR TITLE
Fixed: error in console due to empty shopify configuration on logout(#2tky1f3)

### PR DESCRIPTION
**Pull request by disha1202**
### Related Issues

Closes #

### Screenshots of Visual Changes before/after (If There Are Any)

**IMPORTANT NOTICE** - Remember to add changelog entry

### Contribution and Currently Important Rules Acceptance

- [ ] I read and followed [contribution rules](https://github.com/hotwax/job-manager#contribution-guideline)

### CODE REVIEW

File: src/components/Menu.vue [hunk 1]
```diff
@@ -28,7 +28,7 @@
           <ion-label class="ion-text-wrap">
             <p class="overline">{{ instanceUrl }}</p>
             {{ eComStore.storeName }}
-            <p>{{ currentShopifyConfig.name ? currentShopifyConfig.name : currentShopifyConfig.shopifyConfigName }}</p>
+            <p>{{ currentShopifyConfig?.name ? currentShopifyConfig.name : currentShopifyConfig?.shopifyConfigName }}</p>
```
Comment (ymaheshwari1) [inline src/components/Menu.vue:31]: WDYT?
```suggestion
            <p v-if="currentShopifyConfig">{{ currentShopifyConfig.name ? currentShopifyConfig.name : currentShopifyConfig.shopifyConfigName }}</p>
```

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]

File: src/components/Menu.vue (hunk 1 above)
Comment (adityasharma7) [inline src/components/Menu.vue:31]: I think we should set it as empty object by default and this check will not be required then

**Review by adityasharma7 — CHANGES_REQUESTED** [review]

File: src/components/Menu.vue (hunk 1 above)
Comment (disha1202) [reply src/components/Menu.vue]: Set empty object by default to resolve this issue

Comment (disha1202) [reply src/components/Menu.vue]: Set empty object by default to resolve this issue

**Review by adityasharma7 — APPROVED** [review]

**Pull request merged**